blazor/rating/tooltip.md

```diff
@@ -9,9 +9,7 @@ documentation: ug
 
 # Tooltip in Blazor Rating Component
 
-The Blazor Rating component supports tooltips to provide more information about the rating by using the [ShowTooltip](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_ShowTooltip) property.
-
-Below example demonstrates Tooltip of Rating.
+The Blazor Rating component supports tooltips to provide additional information about the rating by using the [ShowTooltip](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_ShowTooltip) property. When the `ShowTooltip` property is set to `true`, a tooltip will be displayed when the user hovers over a rating item. The tooltip can contain any information you want to provide about the rating, such as the rating value or a description of the rating.
```

Comment:
The Blazor rating component supports the addition of tooltips to show additional information in rating items by setting the ShowTooltip property to true. If enabled, the tooltip appears when the user hovers over a rating item.

Reply:
updated as per comments.